docs/configuring-playbook-backup-borg.md

```diff
@@ -35,17 +41,21 @@ matrix_backup_borg_location_repositories:
 matrix_backup_borg_storage_encryption_passphrase: "PASSPHRASE"
 matrix_backup_borg_ssh_key_private: |
 	PRIVATE KEY
+matrix_backup_borg_location_source_directories:
+  - "{{ matrix_base_data_path }}"
```

Comment:
This is the default as per `group_vars/matrix_servers` already, isn't it?

https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/ce675b0e795ffacc39c37d2ca26aa4659509d9e2/group_vars/matrix_servers#L1084-L1085

We may mention the `matrix_backup_borg_location_source_directories` variable, as you're doing below, but asking people to redefine whatever is the default doesn't seem necessary.

Reply:
My bad, I only looked at `main.yml` and saw the empty variable. Please check the updated version now.